### PR TITLE
Create t66y.com.xml

### PR DESCRIPTION
**Pull request by stlwvikx**

### CODE REVIEW

File: src/chrome/content/rules/t66y.com.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="t66y.com">
+	<target host="t66y.com" />
+	<target host="www.t66y.com" />
+
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/t66y.com.xml]: Please add a `securecookie` tag here.

Comment (stlwvikx) [inline src/chrome/content/rules/t66y.com.xml]: It's important?

Comment (gloomy-ghost) [inline src/chrome/content/rules/t66y.com.xml]: It prevents session cookie leaking from http hijack, can be important for users who have an account and never log out.

**Comment by gloomy-ghost** [comment]
Merged, thanks.

**Pull request merged**